generate-box/Vagrantfile

```diff
@@ -118,6 +119,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   end
 
   config.vm.provision "shell", inline: "apt-get update && apt-get install -y puppet"
+  config.vm.provision "shell", inline: "puppet module install puppetlabs-apt"
```

Comment:
actually, it would be good if that would work, but it does not and we would need still the puppet modules as a copy as part of this repo